git-lfs/git-lfs.install

```diff
@@ -0,0 +1,21 @@
+post_install() {
+  for arch in mingw32 mingw64; do
+    if [ -f "${arch}/bin/git-lfs.exe" ]; then
+      git config --system filter.lfs.clean "git-lfs clean %f"
+      git config --system filter.lfs.smudge "git-lfs smudge %f"
+      git config --system filter.lfs.required true

```

Comment:
Ah, this is the danger of duplicating here, instead of relying on LFS: those values aren't completely correct. If you're bundling LFS v1.2 or lower, then keep this. If not, I'll make sure to add `git lfs install --system` to the next LFS release.

You can fix it for now with:

```
git config --system filter.lfs.clean "git-lfs clean -- %f"
git config --system filter.lfs.smudge "git-lfs smudge -- %f"
```

Here's why we need the `--`: https://github.com/github/git-lfs/pull/1083.